### ISSUE TITLE
Use TOML (or YAML) instead of Markdown for structured data

### ISSUE THREAD

**Issue by frafra** [LivingNorway/LivingNorwayR#23]
Markdown is not meant to be used for structured data. It is great to output simple files, mostly for mixed-content documents (rich text, different headings, tables), but hard to parse it back. It has many dialects and there is no standard.

TOML/YAML/JSON would be preferable. [JSON](https://en.wikipedia.org/wiki/JSON) is less trivial to edit by humans compared to TOML and YAML. YAML has a [problem with "NO" values](https://hitchdev.com/strictyaml/why/implicit-typing-removed/) (potentially common in Norway).

Thus, my suggestion would be to rely on [TOML](https://en.wikipedia.org/wiki/TOML). It is standard, well-structured, easy to parse and strongly typed.

Code example highlighting the issues of using Markdown to store structured data:
https://github.com/LivingNorway/LivingNorwayR/blob/6b62df8772fea2ba0cee65d4c6e9f1a3866517cf/R/read_metadata.R#L8-L14

**Comment by frafra** [comment]
As discussed, the new proposal is to use Markdown with a YAML frontmatter. I will investigate potential technical issues (supported libraries, nested structures, etc.).

**Comment by joechip90** [comment]
We've now overhauled the way we handle metadata so this issue might not be so relevant anymore
